docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/NaiveBayes.tt

```diff
@@ -0,0 +1,24 @@
+﻿<#@ include file="MulticlassClassification.ttinclude"#>
+<#+
+string ClassName="NaiveBayes";
+string Trainer = "NaiveBayes";
+string MetaTrainer = null;
+string TrainerOptions = null;
+
+string OptionsInclude = "";
+string Comments= "";
+
+string ExpectedOutputPerInstance= @"// Expected output:
+            //   Label: 1, Prediction: 2
+            //   Label: 2, Prediction: 2
+            //   Label: 3, Prediction: 2
+            //   Label: 2, Prediction: 2
+            //   Label: 3, Prediction: 2";
+
+string ExpectedOutput = @"// Expected output:
+            // Expected output:
+            // Micro Accuracy: 0.35
```

Comment:
> 0.35 [](start = 30, length = 5)

can we get something above 60%

Reply:
I tried, we cant, this is a linear model :)

---
In reply to: [271938623](https://github.com/dotnet/machinelearning/pull/3159#discussion_r271938623) [](ancestors = 271938623)